src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -219,7 +220,7 @@ function BulkEmailForm(props) {
       <p>{intl.formatMessage(messages.bulkEmailTaskAlertRecipients, { subject: editor.emailSubject })}</p>
       <ul className="list-unstyled">
         {editor.emailRecipients.map((group) => (
-          <li key={group}>{group}</li>
+          <li key={group}>{getDisplayTextFromRecipient(group)}</li>
```

Comment:
This is not going to be internationalized, if I'm understanding this correctly.

You need to either:
1. Make `getDisplayTextFromRecipient(group)` return a `<FormattedMessage />` component, or
2. Make `getDisplayTextFromRecipient(group)` return a `MessageDescriptor` and then pass that to `intl.formatMessage(...)` as seen three lines above.